source/slang/hlsl.meta.slang

```diff
@@ -2153,7 +2153,7 @@ ${{{{
     {
 }}}}
     __intrinsic_op($(kIROp_CombinedTextureSamplerGetTexture))
-    __TextureImpl<T, Shape, isArray, isMS, sampleCount, 0, isShadow, 0, format> __getTexture();
+    __TextureImpl<T, Shape, isArray, 0, sampleCount, 0, isShadow, 0, format> __getTexture();
```

Comment:
It seems like I made a mistake here.
I wonder how this has been working?
I don't see where `isMS` is declared.

Reply:
I was confused too, not sure why it just compiled prior to this change, must be some other bugs that I didn't have time to track down.